src/edu/ucsd/library/xdre/statistic/analyzer/LogAnalyzer.java

```diff
@@ -31,7 +31,8 @@
  * @author lsitu@ucsd.edu
  */
 public class LogAnalyzer{
-	
+	public static String SPEC_COLL_URL = "http://library.ucsd.edu/speccoll/";
```

Comment:
I've removed the http part of the URL, see commit https://github.com/ucsdlib/damsmanager/pull/168/commits/c0e9653aa7fc56b9108d7fd0285a43009273608b. Thanks.